### DISCUSSION TITLE
[Bug]: cannot render a website after backup recovery

### DISCUSSION THREAD

**Discussion by martinobettucci** [GetPublii/Publii#1397] [Help]
### Operating system

Ubuntu 22.04

### Publii version

0.44.1

### Post editor

None

### Bug description

Le processus de rendu a échoué
An error (1003) occurred during preparing CSS overrides.

Invalid or unexpected token

### Steps to reproduce

Backup a site, reload it, try to pre-render

**Comment by dziudek** [comment]
Hi,

Seems to be a bug with your theme/setup not like a common issue (for me backups works fine). At this moment I am converting this issue to discussion.

**Reply by dziudek** [reply]
@martinobettucci - most probably your file visual-override.js in the theme is corrupted or contains invalid syntax
